### ISSUE TITLE
Tube model wrong output

### ISSUE THREAD

**Issue by Bk8** [SMCFY/SMC-WDF#1]
I was reading the paper: Wave Digital Simulation of a Vacuum-Tube Amplifier recreated in the TubeModel.m and something is wrong or missing, the model it is not behaving like a tube. Since Vg = 20 is constant the tube is always saturated, and  is not posible to recreate Fig. 5 in the paper where saturation is a directly related to the amplitude. Also the implementation is weird, since the signal is feeded from a constant voltage source.

**Comment by NikolajAndersson** [comment]
Yes, I remember the response not being correct and behaving weird. Feel free to update it and make a pull request.

**Comment by kevontheweb** [comment]
@Bk8 did you ever solve this? I'm trying to reimplement this in python and have the same issue.

**Comment by Bk8** [comment]
@kevin-nel No, I was not able to fix it. There are a couple of errors, not even the Newton-Raphson solver is correctly implemented. Maybe if you have a public repo of the Python implementation I could take a look again at it.
